apps/web/src/components/ConnectWalletButton/ConnectWalletButton.tsx

```diff
@@ -54,6 +56,9 @@ export function ConnectWalletButton({
     () => switchChain({ chainId: base.id }),
     [switchChain],
   );
+  const searchParams = useSearchParams();
+  const showChainSwitcher = searchParams?.get('showChainSwitcher');
```

Comment:
can we make this contingent on process.env.NODE_ENV === 'development' as well and/or instead?

Reply:
we need this in production speficially.